src/Afsy/Bundle/FrontBundle/Resources/views/Avent/day_2017_05.html.twig

```diff
@@ -0,0 +1,558 @@
+        correspondant à l’image que vous utilisez et remplacer le <code>FROM php:7.0</code>
+        par le contenu du fichier original. Pour moi il s’agit de PHP FPM 7.0 basé sur
+        debian jessie (si vous ne savez pas quoi prendre entre debian et alpine, prenez debian).
+        Pour moi il s’agit donc de ce <a href="Nous allons donc devoir tricher. L’astuce consiste à copier/coller le Dockerfile correspondant à l’image que vous utilisez et remplacer le `FROM php:7.0` par le contenu du fichier original. Pour moi il s’agit de PHP FPM 7.0 basé sur debian jessie (si vous ne savez pas quoi prendre entre debian et alpine, prenez debian). Pour moi il s’agit donc de ce Dockerfile:
+https://github.com/docker-library/php/blob/0bb4068bd639ba98631bc2999e0d20cae583ec00/7.0/jessie/fpm/Dockerfile
+">Dockerfile</a>. Vous trouverez la liste de touts les Dockerfile PHP à cette adresse :
+        <a href="https://github.com/docker-library/php">https://github.com/docker-library/php</a>
+    </p>
+
+    <p>Après l’avoir copié, il va falloir le modifier pour l’adapter à nos besoins :</p>
+    <ul>
+        <li>Ajouter l’option debug à la compilation de PHP</li>
+    </ul>
+
+    <p>Voici donc à quoi ressemble ma commande <code>./configure</code> dans le Dockerfile :</p>
+    <pre><code class="language-bash">./configure \
+     --build="$gnuArch" \
+     --with-config-file-path="$PHP_INI_DIR" \
+     --with-config-file-scan-dir="$PHP_INI_DIR/conf.d" \
+     \
+     --disable-cgi \
+     \
+# --enable-ftp is included here because ftp_ssl_connect() needs ftp to be compiled statically (see https://github.com/docker-library/php/issues/236)
+     --enable-ftp \
+# --enable-mbstring is included here because otherwise there's no way to get pecl to use it properly (see https://github.com/docker-library/php/issues/195)
+     --enable-mbstring \
+# --enable-mysqlnd is included here because it's harder to compile after the fact than extensions are (since it's a plugin for several extensions, not an extension in itself)
+     --enable-mysqlnd \
+     \
+     --with-curl \
+     --with-libedit \
+     --with-openssl \
+     --with-zlib \
+     --enable-debug \         # <----- On a simplement rajouté ceci !
+     \
+# bundled pcre is too old for s390x (which isn't exactly a good sign)
+# /usr/src/php/ext/pcre/pcrelib/pcre_jit_compile.c:65:2: error: #error Unsupported architecture
+     --with-pcre-regex=/usr \
+     --with-libdir="lib/$debMultiarch" \
+     \
+     $PHP_EXTRA_CONFIGURE_ARGS</code></pre>
+    <p>Le point important est la ligne <code>--enable-debug</code>.</p>
+
+    <ul>
+        <li>Copier/coller les scripts nécessaires à l’exécution du Dockerfile (les fichiers docker-php-*) depuis le repository dans lequel vous avez récupéré le Dockerfile</li>
+        <li>Commenter/supprimer la ligne qui supprime les symboles (évidemment!)</li>
+    </ul>
+    <pre><code class="language-bash">    && make install \
```

Comment:
Better making sure that all special characters are converted into html entities :)